README.md

```diff
@@ -7,6 +7,6 @@ DiscordAuth provides OAuth-based identity verification services for MUN-affiliat
 
 ## Usage
 
-Simply redirect a user to `https://auth.muncompsci.ca`.
+Simply redirect a user to `https://discord.muncompsci.ca`.
```

Comment:
isn't this `https://discord.muncompsci.ca/auth`?

Reply:
Fixed